project/AppDependencies.scala

```diff
@@ -22,11 +22,13 @@ object AppDependencies {
 
   private val urlBuilderVersion = "3.1.0"
   private val ibm4jVersion = "63.1"
+  private val frontendBootstrapVersion = "12.9.0"
 
   def compile(): Seq[ModuleID] = Seq(
     "com.typesafe.play" %% "play" % PlayVersion.current % "provided",
     "uk.gov.hmrc" %% "url-builder" % urlBuilderVersion,
-    "com.ibm.icu" % "icu4j" % ibm4jVersion
+    "com.ibm.icu" % "icu4j" % ibm4jVersion,
+    "uk.gov.hmrc" %% "frontend-bootstrap" % frontendBootstrapVersion
```

Comment:
change this for bootstrap-play-25 so we can cross-compile it for play 26 easily. Shouldnt be too many services relying on frontend bootstrap for too much longer.

Reply:
applied comment